src/SignalR/clients/ts/signalr/src/browser-index.ts

```diff
@@ -16,8 +16,7 @@ if (!Uint8Array.prototype.indexOf) {
 }
 if (!Uint8Array.prototype.slice) {
     Object.defineProperty(Uint8Array.prototype, "slice", {
-        // wrap the slice in Uint8Array so it looks like a Uint8Array.slice call
-        value: (start?: number, end?: number) => new Uint8Array(Array.prototype.slice(start, end)),
+        value: Array.prototype.slice,
```

Comment:
> Basically the "this" wasn't passed through to our polyfill so we were just calling Array.prototype.slice which would return an empty array every time.

Why not just bind Array.prototype.slice to this?

```suggestion
        value: (start?: number, end?: number) => new Uint8Array(Array.prototype.slice.bind(this)(start, end))
```

Reply:
If you know how then please share, but I've tried many different things and none work.

Reply:
...